server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/ProjectService.java

```diff
@@ -66,4 +77,55 @@ public ProjectService(ProjectManager projectManager,
         final ProjectDto dto = Jackson.readValue(message.content().toStringAscii(), ProjectDto.class);
         return execute(Command.createProject(author, dto.getName())).thenApply(unused -> dto);
     }
+
+    // TODO(hyangtack) Test APIs to manipulate metadata.
+    //                 These test APIs would be merged into existing HTTP APIs.
+    @Get("/mds/projects")
+    public CompletionStage<List<ProjectInfo>> listProjectInfo() {
+        return mds.findProjects(AuthenticationUtil.currentUser());
+    }
+
+    @Post("/mds/projects/{projectName}")
+    public CompletionStage<ProjectInfo> createProjectInfo(@Param("projectName") String projectName) {
```

Comment:
I think this should be called when a project is created. But let's do it when this is merged with the HTTP API pr.

Reply:
As I mentioned above these APIs, they were added for testing purpose. Also it should be merged into the existing HTTP APIs later.